atlasdb-cassandra/src/main/java/com/palantir/atlasdb/cassandra/CassandraKeyValueServiceConfig.java

```diff
@@ -298,7 +297,7 @@ default String type() {
     @Override
     @Value.Default
     default int concurrentGetRangesThreadPoolSize() {
-        return poolSize() * servers().size();
+        return poolSize() * servers().thrift().size();
```

Comment:
yeah, I think definitely add a `numberOfHosts` field, that's `@Value.Derived` on each of the impls.

Reply:
I think it this case it does make more sense to have a derived value but I would call it, at least at the moment, `numberOfThriftHosts` as this config entry is currently specific for Thrift related code.